.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: Lint,Test,Deploy
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Node setup
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+      - run: npm ci
+      - run: npm run lint
```

Comment:
Please create `npm run (lint/test/deploy):affected` commands and use it instead.

Reply:
hi @NetanelBasal pls can you check the new changes if it's cool